apps/client/src/app/export/export.component.html

```diff
@@ -77,6 +78,7 @@ <h1>Export</h1>
             <label>Output Format</label>
             <select class="form-control" [(ngModel)]="options.responseFormat" (change)="responseFormatChanged()">
               <option value="application/json">JSON</option>
+              <option value="application/msword">MSWORD</option>
```

Comment:
See comment above. MS Word should be separate tab, not an option under "Output Format".

Reply:
Completed